data/ignition.template

```diff
@@ -124,3 +125,18 @@ storage:
 
           [Install]
           WantedBy=multi-user.target default.target
+    - path: /etc/systemd/system/agent_id.service
+      mode: 0644
+      contents:
+        inline: |
+          [Unit]
+          Description=Service to retrieve system uuid
+
+          [Service]
+          Type=oneshot
+          ExecStart=/bin/bash -c 'dmidecode -s system-uuid > /var/home/core/agent_id'
```

Comment:
We don't mount $HOME to the container. We mount `/var/home/core/.migration-planner` directory, so would be better to create the file there. Maybe here `/var/home/core/.migration-planner/data`.

Reply:
forgot about the container. you're right.

Reply:
and also dmidecode needs to be installed first. maybe it's simple to cat the system-uuid file directly

Reply:
Yep, let's just use `cat`.